sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -185,17 +185,28 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     }
   }
 
+  private def checkPushedInfo(df: DataFrame, expectedPlanFragment: String): Unit = {
+    df.queryExecution.optimizedPlan.collect {
+      case _: DataSourceV2ScanRelation =>
+        checkKeywordsExistsInExplain(df, expectedPlanFragment)
+    }
+  }
+
   test("simple scan with top N") {
     val df1 = spark.read
       .table("h2.test.employee")
       .sort("salary")
       .limit(1)
     checkPushedLimit(df1, Some(1), createSortValues())
```

Comment:
Seems we don't need these methods anymore? `checkPushedInfo` can cover all of them